source/doi_background.rst

```diff
@@ -1,72 +1,49 @@
+Making a resource findable is the first step to allow for sharing of data, and for this reason this is a key point for achieving FAIRness of data. 
+The first FAIR guideline principle states the following: 
 
-Assigning a PID to a digital resource adheres with the first principle of FAIRness, as defined in XXX. 
-Making a resource findable is the first step to allow for sharing of data. 
-
-Accessibility 
--------------
-
-Interoperability
-----------------
-
-Reusability
------------
-Reusability of data is strictly related with the 
-
-
-Creating a Dataset
-==================
+- F1. (meta)data are assigned a globally unique and persistent identifier.
 
+Thus, assigning a DOI to a digital resource adheres with the first principle of FAIRness, as the identifier will be globally unique as well as persistent.
 
-Supplying the right Metadata
-----------------------------
+Additionally, the third principle for findability: 
 
-Delivering ACDD compliant metadata
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Compiling the metadata collection webform
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+- F3: Metadata clearly and explicitly include the identifier of the data they describe.
 
+points out that metadata and the dataset should be connected explicitly by mentioning a dataset’s globally unique and persistent identifier in the metadata of that 
+specific dataset. This applies more particularly for data formats that do not include metadata in their headers, usually resulting in separate files for data and
+metadata. 
 
-Creation of MMD files and storage
-=================================
-
-Once a data provider has requested a DOI for a specific dataset, metadata information are extracted. Creation of an xml file containing mmd entries can 
-be extracting using the nc-to-mmd.pl tool when netCDF/ACDD compliant files are provided. or by exporting the metadata collection form in an xml file 
-
-
-Creation of landing pages and assignment of DOI
-===============================================
-
-Landing pages for datasets which require a DOI will be automatically created on the https://adc.met.no/[adc.met.no] data portal starting from the 
-available metadata, upon request from the data provider. Only administrators of the data portal has access to this service and are thus the only ones 
-who can finalize the process of assigning DOI. 
-
-A Drupal 7 module has been developed, implemented and configured on the https://adc.met.no/[adc.met.no] data portal. The source code with detailed explanation 
-on how to implement this solution is openly available at: 
-
-https://github.com/ferrighi/landing_page_creator
+Reusability
+-----------
+Reusability of data is strictly related with the possibility of citing data. When the DOI is incorporated into a citation it becomes a guaranteed location for the 
+data, because the DOI will always resolve to the right web address (URL). Using a DOI allows then for proper citation of data when it is reused, giving the right 
```

Comment:
When registering DOI with fex. DataCite, they create a permanent record for that ID that is guaranteed to resolve to the provided URL (which is editable by the one, who has provided it, is that correct?)
What does DataCite do when the provided URL is no longer reachable or when the landing page it links to doesn't provide any further links to the actual data?

Reply:
DOI landing pages does not necessarily have link to data. What you put on the landing page is entirely up to you. It is not DataCite responsibility to expose content. 
For the not reachable URL, I am not sure about the details, but there is an official contract between MET and UNIT for minting of DOI.

Reply:
> When the DOI is incorporated into a citation it becomes a guaranteed location for the 
data

I am worried that those reading the documentation might misinterpret the following as that DOI provides some kind of security when it comes to the actual data, while it really only provides a resolvable link and some metadata minimum.

Reply:
I'll rephrase this into "a guaranteed location for the resource cited"